diff --git a/node_modules/sitemap/dist/lib/sitemap-stream.js b/node_modules/sitemap/dist/lib/sitemap-stream.js
index 6180a01..8940075 100644
--- a/node_modules/sitemap/dist/lib/sitemap-stream.js
+++ b/node_modules/sitemap/dist/lib/sitemap-stream.js
@@ -1,7 +1,6 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.streamToPromise = exports.SitemapStream = exports.closetag = exports.stylesheetInclude = void 0;
-const url_1 = require("url");
 const stream_1 = require("stream");
 const types_1 = require("./types");
 const utils_1 = require("./utils");
@@ -9,8 +8,6 @@ const sitemap_item_stream_1 = require("./sitemap-item-stream");
 const errors_1 = require("./errors");
 const xmlDec = '<?xml version="1.0" encoding="UTF-8"?>';
 const stylesheetInclude = (url) => {
-    // Throws if url is invalid
-    new url_1.URL(url);
     return `<?xml-stylesheet type="text/xsl" href="${url}"?>`;
 };
 exports.stylesheetInclude = stylesheetInclude;
